addons/netfox.extras/state-machine/network-state.gd

```diff
@@ -0,0 +1,33 @@
+@icon("res://addons/netfox/icons/state-synchronizer.svg")
+extends Node
+class_name NetworkedState
+
+@export var state_machine: NetworkedStateMachine
```

Comment:
I think we could just not `@export` this property and always grab the parent in `_ready`.